frontend/src/views/App/AppFooter.tsx

```diff
@@ -156,24 +151,18 @@ export default function AppFooter() {
                 <Link href="/contact">Contact</Link>
               </li>
               <li className={classes.listItem}>
-                <Link href="/impressum">Impressum</Link>
+                <Link href="https://heigit.org/imprint/">Impressum</Link>
```

Comment:
I think "Impressum" is the German word, "imprint" would be the correct word in English right?

Reply:
Suggestion: Change the text of the link to "imprint" aswell :)